src/get-definitely-typed.ts

```diff
@@ -43,10 +44,18 @@ if (!module.parent) {
     });
 }
 
-export async function getDefinitelyTyped(options: Options): Promise<FS> {
+/**
+ * Return the commit SHA1 of the master tip of DefinitelyTyped
+ */
+export async function resolveDefinitelyTypedMaster(githubAccessToken: string, fetcher: Fetcher): Promise<string> {
+    const masterRef = await queryGithub("repos/DefinitelyTyped/DefinitelyTyped/git/refs/heads/master", githubAccessToken, fetcher) as { object: { sha: string } };
```

Comment:
Is master the correct SHA to use here? In particular, it's possible that it may have moved between the time that this package was merged. Another package may have been merged in between.

I think it would be better if you use the latest sha just for the package's directory.

Reply:
It doesn't really matter - this SHA will be used to download the zip, so it is guaranteed that `gitHead` points to the exact revision that is being published. That is all that should matter.

It would be nice, but I wouldn't know where to get that commit from. Can you point to where?

Reply:
Here’s the code I used, although that was using octokit. The rest call shouldn’t be too different though. 

https://github.com/sandersn/types-publisher-watchdog/commit/f6471155a26702692e8abcd82c001c408e78354c#diff-168726dbe96b3ce427e7fedce31bb0bcL76

Look at the part I deleted starting with `const oops`

Reply:
Ah, so you would get the latest commit from the git log of that folder. I thought you were suggesting to use the commit ID from the triggering event, e.g. the commit hash of the merge commit of a PR that was merged which triggered the new publish